modules/installation-mirror-repository.adoc

```diff
@@ -52,14 +53,16 @@ registry, such as `ocp4/openshift4`.
 the pull secret for your mirror registry that you created.
 <6> The release mirror. For a production release, you must specify
 `ocp-release`.
+<7> For `server_architecture`, specify the architecture of the server, such as `x86_64`.
+
 
 . Mirror the repository:
 +
 ----
 $ oc adm -a ${LOCAL_SECRET_JSON} release mirror \
-     --from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE} \
+     --from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} \
      --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} \
-     --to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}
+     --to-release-image=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE}
```

Comment:
@kalexand-rh The following step 5 missed ${ARCHITECTURE}

Reply:
Fixed. Thanks!